src/main/java/subway/path/domain/SubwayGraph.java

```diff
@@ -0,0 +1,14 @@
+package subway.path.domain;
+
+import org.jgrapht.graph.WeightedMultigraph;
+
+public class SubwayGraph<V, E> extends WeightedMultigraph {
+    public SubwayGraph(Class edgeClass) {
```

Comment:
Generic의 올바른 이용:
```suggestion
public class SubwayGraph<V, E> extends WeightedMultigraph<V, E> {
    public SubwayGraph(Class<E> edgeClass) {
```
또는, SubwayGraph는 이 프로젝트에서 항상 <Station, SubwayEdge> 라고 보이는데 그러면 차라리
```suggestion
public class SubwayGraph extends WeightedMultigraph<Station, SubwayEdge> {
    public SubwayGraph() {
        super(SubwayEdge.class);
    }
```
처럼 해도 되지 않을까요?

Reply:
Generic에 대해 아직 미숙했는데 이 참에 한번 찾아봐야겠네요.
SubwayGraph 클래스는 SubwayWeight와 같이 굳이 따로 구현할 필요가 없을 것 같아서 삭제했습니다.